### PR TITLE
modify equalLengths function

### PR DESCRIPTION
**Pull request by mvnnn**
issue #9313

### CODE REVIEW

**Comment by mvnnn** [comment]
Can anyone review this PR?

File: sympy/printing/pretty/stringpict.py [hunk 1]
```diff
@@ -38,11 +38,25 @@ def __init__(self, s, baseline=0):
...
     def equalLengths(lines):
         # empty lines
+        _max_dot = 0
         if not lines:
             return ['']
 
         width = max(len(line) for line in lines)
-        return [line.center(width) for line in lines]
+
+        if any('v̇' in s for s in lines):
```
Comment (asmeurer) [inline sympy/printing/pretty/stringpict.py]: Why does this special-case `v̇`?

Comment (mvnnn) [inline sympy/printing/pretty/stringpict.py]: length of 'v̇' is 2 but it's need space is equal to one character.

**Comment by oscargus** [comment]
A more general approach is provided in #12205 so I close this for now.

**Pull request closed without merging**